index.mjs

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+import { S3Client, GetObjectCommand, PutObjectCommand } from '@aws-sdk/client-s3';
+import { Response } from 'node-fetch';
+
+
+const s3Client = new S3Client({region: 'us-east-2'});
+
+export const handler = async(event) => {
+  console.log('Event is live!', event);
+    
+  const Bucket = '';
+  const Key ='images.json';
+  console.log('Bucket and Key here', { Bucket, Key });
+
+  let file = event.Records[0].s3.object.key;
+  let size = event.Records[0].s3.object.size;
+  let type = 'jpg';
```

Comment:
Just a minor thing, what if the image isn't a jpg? Can we handle different file extensions?